modules/github_integration/frontend/module/tab-header/tab-header.template.html

```diff
@@ -0,0 +1,30 @@
+<div class="github-pr-header">
+  <h3 class="title">
+    <op-icon icon-classes="button--icon icon-merge-branch"></op-icon>
+    {{text.title}}
+  </h3>
+  <ul class="toolbar-items hide-when-print">
+    <li class="toolbar-item">
+      <button class="button github-create-pr"
+          [disabled]="true"
+          [attr.aria-label]="text.createPrButtonDescription">
```

Comment:
It's usually good form to make buttons `type="button"`. They're `type="submit"` by default, and if this tab header gets included into a larger form switching tabs might submit the form by accident.

Reply:
I made all buttons `type="button"`, thanks for the recommendation :)